serverless/knative_eventing/serverless-using-brokers.adoc

```diff
@@ -1,16 +1,18 @@
 [id="serverless-using-brokers"]
-= Using brokers with Knative Eventing
+= Using brokers as an event delivery mechanism
 include::modules/common-attributes.adoc[]
 include::modules/serverless-document-attributes.adoc[]
 :context: serverless-using-brokers
 
 toc::[]
 
 Knative Eventing uses the `default` broker unless otherwise specified.
+You can create this broker by using the `knative-eventing-injection` annotation.
```

Comment:
Will you please review and clarify this intro? I'm not sure if we're making a broker or updating the default broker, and at this point, I'm unsure what the importance of the `knative-eventing-injection` annotation is.